.github/workflows/academic_validation.yml

```
@@ -0,0 +1,417 @@
+    - name: Download all test results
+      uses: actions/download-artifact@v3
+      with:
+        path: all_results/
+    
+    - name: Assess Quality Gates
+      run: |
+        python -c "
+        import json
+        import sys
+        from pathlib import Path
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)